src/core/org/apache/jmeter/util/JSR223TestElement.java

```diff
@@ -86,11 +88,16 @@ public JSR223TestElement() {
     }
 
     protected ScriptEngine getScriptEngine() throws ScriptException {
-        final String lang = getScriptLanguage();
+        String lang = getScriptLanguage();
+        
+        if (lang.isEmpty()) {
+            lang = defaultScriptLanguage;
+            log.warn("Script language has not been chosen on the UI, the script will be interpreted as a groovy script");
+        }
 
         ScriptEngine scriptEngine = getInstance().getEngineByName(lang);
         if (scriptEngine == null) {
```

Comment:
Hi Milamber,

I have keep the exception because I am not sure that getInstance().getEngineByName("groovy") will always return something.
For example if the groovy jar are missing

Antonio